src/Phoenix.vue

```diff
@@ -41,31 +35,7 @@ export default {
     ...mapGetters(['configuration']),
     showHeader () {
       return this.$route.meta.hideHeadbar !== true
-    },
-    showBackground () {
-      if (!this.$route.meta.showBackground) {
```

Comment:
showBackground is on some route definitions as well - please remove them - no longer needed